modules/nf-core/kmcp/profile/main.nf

```diff
@@ -2,15 +2,14 @@ process KMCP_PROFILE {
     tag "$meta.id"
     label 'process_medium'
 
-    conda "bioconda::kmcp=0.9.1"
+    conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/kmcp:0.9.1--h9ee0642_0':
-        'biocontainers/kmcp:0.9.1--h9ee0642_0' }"
+        'https://depot.galaxyproject.org/singularity/kmcp:0.9.4--h9ee0642_0':
+        'biocontainers/kmcp:0.9.4--h9ee0642_0' }"
```

Comment:
are you tracking version updates of tools in the changelog normally?

Reply:
Yes, in the table under 'depdnecnies'. Borrowed it from mag as I thought it was a nice idea :)